libs/s25main/desktops/dskGameInterface.cpp

```diff
@@ -930,8 +930,8 @@ void dskGameInterface::Run()
 
 void dskGameInterface::GI_StartRoadBuilding(const MapPoint startPt, bool waterRoad)
 {
-    // Im Replay und in der Pause keine Straßen bauen
-    if(GAMECLIENT.IsReplayModeOn() || GAMECLIENT.IsPaused())
+    // Im Replay keine Straßen bauen
+    if(GAMECLIENT.IsReplayModeOn())
```

Comment:
I changed this because I felt it fits better with the behaviour of the other controls. (i.e. you can also look at which buildings to build or at options windows, etc.) And since I added a check that you can only finish building a road outside of a pause this didn't feel necessary anymore.

But if you prefer to continue disallowing this specifically I can reinstate the old version of this line instead

Reply:
Agreed, ok